### ISSUE TITLE
Add Support for the `source_image` field on `additional_disks`

### ISSUE THREAD

**Issue by agosto-lorinronken** [terraform-google-modules/terraform-google-vm#249]
### TL;DR

Please add support for specifying the `souce_image` field when configuring `additional_disks` in your machine image template.   This can be accomplished by adding the `source_image` field in the `variables.tf` file like so:

```
variable "additional_disks" {
  description = "List of maps of additional disks. See https://www.terraform.io/docs/providers/google/r/compute_instance_template.html#disk_name"
  type = list(object({
    disk_name    = string
    device_name  = string
    auto_delete  = bool
    boot         = bool
    disk_size_gb = number
    disk_type    = string
    disk_labels  = map(string)
    source_image = string
  }))
  default = []
}
`````

### Additional information

This should be a relatively simple change.  I have found that there are occasions when we need to specify a `source_image` for specific additional disks that we are adding to our machine image template.  Terraform supports this as seen in the documentation [here](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance_template#source).
